components/color-picker/color-format.component.ts

```diff
@@ -39,7 +39,7 @@ import { NzColorPickerFormatType, ValidFormKey } from './typings';
   selector: 'nz-color-format',
   exportAs: 'nzColorFormat',
   changeDetection: ChangeDetectionStrategy.OnPush,
-  imports: [ReactiveFormsModule, NzSelectModule, NzInputDirective, NzInputGroupComponent, NzInputNumberComponent],
+  imports: [ReactiveFormsModule, NzSelectModule, NzInputDirective, NzInputWrapperComponent, NzInputNumberComponent],
```

Comment:
I prefer to import ngModule

```suggestion
  imports: [ReactiveFormsModule, NzSelectModule, NzInputModule, NzInputNumberModule],
```